src/main/java/picard/illumina/parser/readers/BaseBclReader.java

```diff
@@ -16,16 +16,32 @@
 public class BaseBclReader {
     private static final byte BASE_MASK = 0x0003;
     private static final byte[] BASE_LOOKUP = new byte[]{'A', 'C', 'G', 'T'};
-    private static final byte NO_CALL_BASE = (byte) '.';
+    public static final byte NO_CALL_BASE = (byte) 'N';
```

Comment:
I think this will break basecalls to sam.

Reply:
Ugh, it does indeed break the tests.  Though it seems odd to me that `IlluminaBasecallsToSam` outputs periods in the reads.  I'm tempted to just update the test cases to Ns.  Thoughts?

Reply:
And round-tripping to BAM and back converts those to Ns.

Reply:
I think that is fine. I'm not sure if there are any additional picard tests that may break in other places (private). 

I'd also worry a bit about anyone that has written similar type integration tests using basecalls to sam and doing direct comparisons